content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md

```diff
@@ -484,16 +484,18 @@ Laboratory equipment, Computer vision
 ### Recommended Operating Conditions
 
 | Symbol | Description                                      | Min             | Max            |
-| ------ | ------------------------------------------------ | --------------- | -------------- |
+|--------|--------------------------------------------------|-----------------|----------------|
 |        | Conservative thermal limits for the whole board: | -40 °C (-40 °F) | 85 °C (185 °F) |
 
 ### Power Consumption
 
-| Symbol  | Description                              | Min  | Typ  | Max  | Unit |
-| ------- | ---------------------------------------- | ---- | ---- | ---- | ---- |
-| VINMax  | Maximum input voltage from VIN pad       | -0.3 | -    | xx   | V    |
-| VUSBMax | Maximum input voltage from USB connector | -0.3 | -    | xx   | V    |
-| PMax    | Maximum Power Consumption                | -xx  | -    | xx   | mW   |
+| Description                                                 | Min | Typ  | Max | Unit |
+|-------------------------------------------------------------|-----|------|-----|------|
+| Power consumption in deep sleep mode with 3.7V power supply |     | 0.67 |     | mA   |
+| Power consumption in busy loop with 5V power supply         |     | 2.6  |     | mAh  |
+| Maximum power consumption with 5V power supply              |     | 230  |     | mA   |
```

Comment:
According to the power tree, the PF1550 has a LDO with a 2A output.
![image](https://user-images.githubusercontent.com/75624145/215080202-8a1f8960-c848-4fa3-97bb-157a935b0f3d.png)
With a 5V input and at 2A output current of the LDO, then the maximum consumption by the board would be at least 2000mA. 
Could we specify what is meant by maximum in this context?

Reply:
Explained better